portfolio/src/main/webapp/index.html

```diff
@@ -10,19 +10,23 @@
     <div id="content">
       <h1>Jess's Portfolio</h1>
       
-      <img src="images/JessDeJong-photo.png" alt="Profile photo with my twin" height="250">
-      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
 
-      <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>. 
-      However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>, 
-      a <strong>CS and Math Double Major</strong>, a <strong>fraternal twin</strong>, an 
-      <strong>amateur unicyclist</strong>, and a <strong>tennis enthusiast</strong>. When I work, 
-      I describe myself as a <strong>curious developer</strong>. In the future, I hope to become 
-      a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a 
-      <strong>full-time software developer</strong>.
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div class="speech-bubble">
+        <div class="arrow"></div>
+
+        <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>. 
+        However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>, 
```

Comment:
nit: `<strong>` is not equivalent to `<b>` (bold tag). `<strong>` has more semantic meaning and is not often used. You should use `<b>` instead.

If we're getting _really_ fancy, you would put these in a `<span>` and style it via a CSS class using the `font-weight` property.

Reply:
Thanks, I'll make sure to fix this and use <b> tags in the future.

Reply:
That was supposed to say < b>. Haha